xrpl/models/requests/request.py

```diff
@@ -150,7 +159,7 @@ def get_method(cls: Type[Request], method: str) -> Type[Request]:
         if method in request_methods:
             return request_methods[method]
 
-        raise XRPLModelException(f"{method} is not a valid Request method")
+        return request_models.CustomRequest
```

Comment:
please update teh doc string. this method no longer raises and it will return `CustomRequest` if no suitable model is found.